setup.py

```diff
@@ -38,22 +38,12 @@ def alltests():
 
 setup(
     name = 'z3c.schema',
-    version='1.0.1.dev0',
+    version=read('version.txt').strip(),
```

Comment:
BTW can zest.release handle this?  I remember it can handle version being in a separate file, but I don't remember what filenames it looks for.